src/docs/development/tools/android-studio.md

```diff
@@ -129,31 +132,56 @@ of your changes almost instantly with the _hot reload_ feature. See
 
 ### Debugging visual layout issues
 
-To debug a visual issue, start the app with **Debug**, and then open the Flutter
-inspector tool window using **View > Tool Windows > Flutter Inspector**.
+To debug a visual issue, start the app with **Debug**,
+and then open the Flutter inspector tool window using
+**View > Tool Windows > Flutter Inspector**.
 
 ![Flutter Inspector Window]({% asset tools/android-studio/visual-debugging.png @path %})
 
 This offers many debugging tools; for details on these see
 [Debugging Flutter Apps](/docs/testing/debugging).
 
-* **Toggle Select Widget Mode**: Select a widget on the device to inspect it in the
-  [Flutter Inspector](/docs/development/tools/inspector).
-* **Toggle Debug Paint**: Add visual debugging hints to the rendering displaying
-   borders, padding, alignment, and spacers.
-* **Toggle Platform Mode**: Toggle between rendering for Android or iOS.
-* **Toggle Performance Overlay**: Show performance graphs for the GPU & CPU threads.
-* **Open Timeline View**: Analyze activity of the application as it runs.
-* **Open Observatory**: A profiler for Dart applications.
-
-Also available in the additional actions menu:
-
-* **Show Paint Baselines**: Causes each RenderBox to paint a line at each of its
-  baselines.
-* **Enable Repaint Rainbow**: Shows rotating colors on layers when repainting.
-* **Enable Slow Animations**: Slows down animations to enable visual inspection.
-* **Hide Debug Banner**: Hides the debug banner even when running a debug build.
-
+* **Enable select widget mode**: Select a widget on the device to inspect
+  in the [Flutter Inspector](/docs/development/tools/inspector).
+* **Refresh widget info**: Redisplay the current widget info.
+* **Show/hide performance overlay**: Toggle display of performance graphs
+  for the GPU & CPU threads.
+* **Toggle platform mode**: Toggle between rendering for Android or iOS.
+* **Show debug paint**: Add visual debugging hints to the rendering
+  displaying borders, padding, alignment, and spacers.
+* **Show paint baselines**: Cause each RenderBox to paint a line at each
+  of its baselines.
+* **Enable slow animations**: Slow down animations to enable visual
+  inspection.
+
+Also available in the "more actions" menu:
+
+* **Show Repaint Rainbow**: Shows rotating colors on layers when repainting.
+* **Hide Debug Mode Banner**: Hides the debug banner even when running a
+  debug build.
+* **Auto horizontal scroll**: XXX - TBD
+* **Highlight nodes displayed in both trees**: XXX - TBD
```

Comment:
this one is highlighting nodes shown in both the details and summary trees in the inspector.